src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -0,0 +1,190 @@
+      color: 'green'} 
+    }}>
+      <IconButton onClick={() => setOpen(!open)}>
+        {open ? <KeyboardArrowUpIcon /> : <KeyboardArrowDownIcon />}
+      </IconButton>
+      <span>{project ? project.name : ""}</span>
+      {open && 
+        <>
+          {loading ? 
+            <Box sx={{textAlign: 'center'}} >
+              <CircularProgress sx={{ scale: "100%", mt: "3%", mb: "3%"}} />
+            </Box>
+            :
+            <DataGrid
+              sx={{
+                backgroundColor: "#f6f6f6",
+                borderTop: 0,
+                borderLeft: 0,
+                borderRight: 0,
+                borderBottom: 0,
+                '& .header-color': {
+                  backgroundColor: '#f2f2f2',
+                }
+              }}
+              autoHeight={true}
+              localeText={{ noResultsOverlayLabel: strings.sprint.notFound }}
+              disableColumnFilter
+              hideFooter={true}
+              filterModel={
+                {
+                items: [{ 
+                  field: 'status',
+                  operator: 'contains', 
+                  value: filter
+                }]
+              }}
+              rows={tasks}
+              columns={[
+                { 
+                  field: 'title',
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.taskName,
+                  minWidth: 0,
+                  flex: 3
+                },
+                { 
+                  field: 'assignedPersons',
+                  headerClassName: 'header-color',         
+                  headerName: strings.sprint.assigned, 
+                  flex: 1
+                },
+                { 
+                  field: 'status',
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.taskStatus, 
+                  flex: 1, valueGetter: (params) => lowerCaseAllWordsExceptFirstLetters(params.row.statusCategory || ''),
+                },
+                { 
+                  field: 'priority',
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.taskPriority,
+                  cellClassName: (params) => params.row.highPriority ? 'high_priority' :  'low_priority',
+                  flex: 1, valueGetter: (params) => params.row.highPriority ? 'High' : 'Normal'   
+                },
+                { field: 'estimate',
+                  headerClassName: 'header-color', 
+                  headerName: strings.sprint.estimatedTime, 
+                  flex: 1, valueGetter: (params) => getHoursAndMinutes(params.row.estimate || 0) 
+                },
+                { 
+                  field: 'timeEntries',headerClassName: 'header-color', 
+                  headerName: strings.sprint.timeEntries, 
+                  flex: 1, valueGetter: (params) => getHoursAndMinutes(getTotalTimeEntries(params.row)) 
+                }
+              ]}
```

Comment:
Again you could have the columns as a seperate variable to keep the render method as "clean" as possible